torch_geometric/nn/conv/cugraph/rgcn_conv.py

```diff
@@ -29,6 +29,8 @@ class CuGraphRGCNConv(CuGraphModule):  # pragma: no cover
     :class:`~torch_geometric.nn.conv.RGCNConv` based on the :obj:`cugraph-ops`
     package that fuses message passing computation for accelerated execution
     and lower memory footprint.
+    The easiest way to enable :obj:`cugraph-ops` is to use
+    https://catalog.ngc.nvidia.com/orgs/nvidia/containers/pyg
```

Comment:
Can we follow the same formatting as other occurrences, i.e.,
```
`"XYZ" <url>`_
```

Also, I think we should add a section in https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html and include both manual installation and the image.

Reply:
so currently theres no available cugraph-ops install except the container, its coming to CUDNN soon and will be on automatically. but i can add something to installation describing this.